app/src/main/java/com/github/muellerma/mute_reminder/MainActivity.kt

```diff
@@ -14,8 +17,28 @@ class MainActivity : AppCompatActivity() {
             "https://github.com/mueller-ma/MuteReminder".openInBrowser(this)
         }
 
-        binding.version.text = getString(R.string.version, BuildConfig.VERSION_NAME)
```

Comment:
This `TextView` should be removed from the xml as well.

Reply:
it has been removed yes

Reply:
Done